core/domain/feedback_services_test.py

```diff
@@ -87,11 +86,13 @@ class SuggestionQueriesUnitTests(test_utils.GenericTestBase):
     def _generate_thread_id(self, unused_exp_id):
         return self.THREAD_ID1
 
-    def _get_threads(self, unused_exploration_id):
-        return self.threads
+    def _put_threads(self):
```

Comment:
Actually -- instead of all this, can we just do "for thread in self.threads:\n thread.put()" on setUp()? Then we don't need to repeat it all the time.